.github/workflows/build.yml

```diff
@@ -0,0 +1,240 @@
+      - name: Run GoReleaser to build Go Pulumi binaries
+        uses: goreleaser/goreleaser-action@v2
+        with:
+          version: latest
+          args: -f .goreleaser.current.yml ${{ inputs.goreleaser-flags }}
+      - name: Upload pulumi-${{ matrix.os }}-arm64
+        uses: actions/upload-artifact@v2
+        with:
+            name: pulumi-${{ matrix.os }}-arm64
+            path: goreleaser/pulumi*-${{ matrix.os }}-arm64*
+            retention-days: 2
+      - name: Upload pulumi-${{ matrix.os }}-x64
+        uses: actions/upload-artifact@v2
+        with:
+            name: pulumi-${{ matrix.os }}-x64
+            path: goreleaser/pulumi*-${{ matrix.os }}-x64*
+            retention-days: 2
+
+  build_python_sdk:
+    name: Build Pulumi Python SDK wheel
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ env.PR_COMMIT_SHA }}
+      - name: Fetch Tags
+        run: |
+          git fetch --quiet --prune --unshallow --tags
+      - name: Install pulumictl
+        uses: jaxxstorm/action-install-gh-release@v1.2.0
+        with:
+          repo: pulumi/pulumictl
+      - name: Set up Python ${{ inputs.python-version }}
+        uses: actions/setup-python@v1
+        with:
+          python-version: ${{ inputs.python-version }}
+      - name: Compute current version to inform wheel build
+        run: |
+          echo "PYPY_VERSION=$(pulumictl get version --language python)" >> $GITHUB_ENV
+      - name: Build Pulumi Python SDK wheel
+        run: |
+            # TODO unify with sdk/python/Makefile once that does not use pipenv
+            cp README.md sdk/python/lib
+            cd sdk/python/lib
+            sed -i.bak "s/\${VERSION}/$PYPY_VERSION/g" setup.py
+            rm setup.py.bak
+            python3 -m venv venv
+            source venv/bin/activate
+            python -m pip install wheel
+            python setup.py build bdist_wheel --python-tag py3
+      - name: Upload pulumi.whl
+        uses: actions/upload-artifact@v2
+        with:
+            name: pulumi.whl
+            path: sdk/python/lib/dist/*.whl
+
+  build_node_sdk:
+    name: Build Pulumi Node SDK tarball
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repo
+        uses: actions/checkout@v2
+        with:
+          ref: ${{ env.PR_COMMIT_SHA }}
+      - name: Fetch Tags
+        run: |
+          git fetch --quiet --prune --unshallow --tags
+      - name: Install pulumictl
+        uses: jaxxstorm/action-install-gh-release@v1.2.0
+        with:
+          repo: pulumi/pulumictl
+      - name: Set up Node ${{ inputs.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ inputs.node-version }}
+      - name: Install yarn
+        run: |
+          npm install -g yarn
+      - name: Make no-op Go command to avoid Go builds
```

Comment:
Is this needed because the builds are linked in the Makefile?

Reply:
Something in the makefile used to try to build Go at this point, but it doesn't seem to do that again. I'll try to remove this.

Reply:
Yes the problem is still there. Something in the makefiles, and I can't find what, is calling `go` command for the Nodejs specific workflow, and performing Go downloads which would be wasteful. I've added some comments about this.